docs/data/toolpad/connecting-to-data-sources/google-sheets.md

```diff
@@ -0,0 +1,29 @@
+# Google Sheets
+
+<p class="description">Google Sheets query type allows us to fetch data from Google sheets living in your Google Drive</p>
```

Comment:
```suggestion
<p class="description">Google Sheets query type allows us to fetch data from Google sheets living in your Google Drive.</p>
```

*x the others*

Reply:
Ok, I think I fixed all the missing dots 🤔